ads/ix.js

```diff
@@ -14,19 +14,39 @@
  * limitations under the License.
  */
 
-import {writeScript} from '../3p/3p';
+import {writeScript, loadScript} from '../3p/3p';
 import {doubleclick} from '../ads/google/doubleclick';
 
+const DEFAULT_TIMEOUT = 500; // ms
+
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
-
 export function ix(global, data) {
   if (!('slot' in data)) {
     global.CasaleArgs = data;
     writeScript(global, 'https://js-sec.indexww.com/indexJTag.js');
   } else { //DFP ad request call
+
+    let calledDoubleclick = false;
+    data.ixTimeout = isNaN(data.ixTimeout) ? DEFAULT_TIMEOUT : data.ixTimeout;
+    const timer = setTimeout(() => {
+      callDoubleclick(global, data);
+    }, data.ixTimeout);
+
+    function callDoubleclick(global, data) {
```

Comment:
can you make it a function variable? also, seems both params are not needed anymore